packages/rx_bloc_cli/lib/src/rx_bloc_cli_constants.dart

```diff
@@ -1,2 +1,11 @@
 /// Indicates the version of the package
-const rxBlocCliPackageVersion = '3.3.1';
+const rxBlocCliPackageVersion = '3.4.0';
+
+/// Generated project's Android Compile SDK version
+const kAndroidCompileSDKVersion = 34;
```

Comment:
Why we need to specify theses settings? Are not they coming from the flutter create command ?

Reply:
By default, the base project generated by the `flutter create` command contains references not to specific values of the android sdk but variables which can be defined within the `local.properties` file (`flutter.compileSdkVersion`,`flutter.targetSdkVersion` and `flutter.minSdkVersion`). If no such variables are defined, they are pulled from the local version of the flutter sdk being in use ([source](https://stackoverflow.com/a/74446579)). 

As the `local.properties` file is checked out of source control, controlling which Android SDK versions the project uses through the mentioned variables can prove to be rather challenging as people now must agree on locally configuring the same variables for each of the developers (Instead of hardcoding the values in the project). Additionally, this can also lead to additional problems when building the app through CI/CD due to same reasons.